remme/clients/account.py

```diff
@@ -64,13 +70,42 @@ def get_account(self, address):
         account.ParseFromString(self.get_value(address))
         return account
 
+    def get_pubkey_account(self, address):
+        pubkey_acc = PubKeyAccount()
+        pubkey_acc.ParseFromString(self.get_value(address))
+        return pubkey_acc
+
     def get_pub_keys(self, address):
+        future_to_address = {}
+        pub_keys = []
         try:
             account = self.get_account(address)
-            return list(account.pub_keys)
         except KeyNotFound:
             return []
 
+        if not account.pub_key_nonce:
+            return pub_keys
+
+        with concurrent.futures.ThreadPoolExecutor(max_workers=5) as executor:
```

Comment:
I understand that you are trying to speed up the program, but this can actually lead to some slowdowns due to GIL and stuff. Furthermore, there is a possibility to retrieve data from multiple addresses at once which is currently not utilized by our wrappers but will be in the near future.

Reply:
Please leave an appropriate FIXME/TODO note here.

Reply:
I could make research about this staff and try to do one request.